### ISSUE TITLE
Smaller Modules

### ISSUE THREAD

**Issue by HaydenReeve** [WASasquatch/was-node-suite-comfyui#111]
Hello! 👋

I thought I'd ask the question before starting to dig through and try and make a few contributions; Is there any consideration to splitting up this giant repo into smaller more manageable functionality scopes?

There's a lot of stuff that I enjoy from this suite—but it's quite hard to manage the sheer quantity of functions and tools made available by a single extension here, especially as many scopes of functionality here seem to have quite large deltas across their usage commonalities.

**Comment by WASasquatch** [comment]
I do not plan to do that, no. I like single file programs and don't like the mess of python. I'd prefer to compile the module in a truly single file that's loaded like a dll if comfy could. 

For me, it's far easier to macro search headers I know about to go anywhere within a single file then flip tabs, open files, etc.

**Comment by HaydenReeve** [comment]
Alright, no sweat. I might look into such a PR at some point in the distant future but for now I appreciate the response!

**Comment by WASasquatch** [comment]
I'm definitely open to it. But use notepad++ like some ancient chode, so it's harder to modularize then it is to just search. Which is also part of why I hadn't when it got so big.

**Comment by HaydenReeve** [comment]
Notepad++ is pretty great, but oh man are you missing out on some nice feature-sets haha.

I've recently started using Jetbrain Fleet to manage my Python AI stuff which feels like a really nice alternative to VSCode or going full PyCharm or whatever.

But fair rationalisation.

I'd definitely recommend looking at Fleet if you had some extra exploratory time, I actually use it to run A1111 and ComfyUI both. Works really well with it's custom launch scripts.

**Comment by WASasquatch** [comment]
I'll take a look at it, thanks. I did not like VSCode at all, also a VSCode PR totally killed colab support cause of typing functions. 

I feel like Stewie Griffin when there's just a hole in the side of the house "I don't like change!" Hahaha why I am so slow to adopt. 

I am hoping to see an AI based python IDE that allows prototyping ideas with Python to fiddle with. I've tried GTP and stuff but man it sucks xD one day.
